pkg/kubelet/server/server.go

```diff
@@ -285,6 +285,7 @@ const pprofBasePath = "/debug/pprof/"
 
 // InstallDeguggingHandlers registers the HTTP request patterns that serve logs or run commands/containers
 func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
+	glog.Infof("Adding Debug Handlers")
```

Comment:
I think `Adding Debug Handlers` is not clear enough. 

Could you please make it more detailed like "Adding debug handlers for kubelet server" or something else more convincing？